DOCUMENTATION.md

```diff
@@ -1672,6 +1673,57 @@ This class allows player movement based on grabbing of `VRTK_InteractableObject`
 
 ---
 
+## Dash Teleporter (VRTK_DashTeleport)
+
+### Overview
+
+The dash teleporter extends the height adjust teleporter and allows to have the `[CameraRig]` dashing to a new teleport location. 
```

Comment:
Just check this doc and make sure you're not putting in unnecessary line endings, you don't need to limit characters per line in this new documentation file :)